arguebuf/models/graph.py

```diff
@@ -708,6 +708,43 @@ def from_ova(
 
         return g
 
+    @classmethod
+    def from_sadface(
+            cls,
+            obj: sadface.Graph,
+            name: t.Optional[str] = None,
+            atom_class: t.Type[AtomNode] = AtomNode,
+            scheme_class: t.Type[SchemeNode] = SchemeNode,
+            edge_class: t.Type[Edge] = Edge,
+            nlp: t.Optional[t.Callable[[str], t.Any]] = None,
+    ) -> Graph:
+        """Generate Graph structure from SADFace argument graph file
+        (reference: https://github.com/Open-Argumentation/SADFace/blob/master/examples/hangback/data.json).
+        """
+        g = cls(name)
+
+        for sadface_node in obj["nodes"]:
+            node = (
+                atom_class.from_sadface(sadface_node, nlp)
+                if sadface_node["type"] == "atom"
+                else scheme_class.from_sadface(sadface_node, nlp)
+            )
+
+            if node:
+                g.add_node(node)
+
+        for sadface_edge in obj["edges"]:
+            if edge := edge_class.from_sadface(sadface_edge, g._nodes):
+                g.add_edge(edge)
+
+        g.metadata = Userdata(obj["metadata"])
+        analyst = Analyst(name=obj["metadata"]["analyst_name"],
```

Comment:
I think it should be `obj["metadata"]["core"]["analyst_name"]` (also for the other fields).